pkg/cmd/server/origin/master.go

```diff
@@ -213,7 +227,7 @@ func (c *MasterConfig) buildHandlerChain(assetConfig *AssetConfig) (func(http.Ha
 				// backwards compatible writer to regular log
 				writer = cmdutil.NewGLogWriterV(0)
 			}
-			handler = apifilters.WithAudit(handler, contextMapper, writer)
+			handler = apifilters.WithLegacyAudit(handler, contextMapper, writer)
```

Comment:
We can do that now (I can take a look) or postpone that to after-merge. You decide.